modules/generator-primer-module/app/templates/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "0.1.0",
```

Comment:
Not sure that this should be minor for the first step. We usually start with `0.0.1` and when it's stable and ready for production we'd bump to a major release `1.0.0`.